Zend/zend_objects.c

```diff
@@ -201,6 +203,10 @@ ZEND_API void ZEND_FASTCALL zend_objects_clone_members(zend_object *new_object,
 			i_zval_ptr_dtor(dst);
 			ZVAL_COPY_VALUE_PROP(dst, src);
 			zval_add_ref(dst);
+			if (has_clone_method) {
+				Z_PROP_FLAG_P(dst) |= IS_PROP_REINITABLE;
```

Comment:
This flag is only used for non-static typed properties, so it would be awesome if someone could help me retrieve the `property_info`.

Reply:
I think `ce->properties_info_table[i]` should do it.